### DISCUSSION TITLE
Change the way we write nlmod in the documentation and publications to NLmod

### DISCUSSION THREAD

**Discussion by rubencalje** [gwmod/nlmod#331] [Polls]
For the documentation-website and publications, I think we should change the name of "nlmod" to "NLmod". This will increase the readability of the name, especially when "nlmod" is at the beginning of a sentence.

In Python-code we would still refer to nlmod in lowercase characters: `import nlmod`

**Comment by raoulcollenteur** [comment]
Is there a publication on NLmod?

**Reply by rubencalje** [reply]
No, just an essay in Dutch about our plans:
https://www.nhv.nu/wp-content/uploads/2022/10/334777_NHV_02_Stromingen-3-2022-ESSAY-CALJE-HR.pdf

**Reply by martinvonk** [reply]
We could write a small article for the Journal of Open Source Software? https://joss.theoj.org/

**Comment by bdestombe** [comment]
My 2 cents. NLMod and NLmod would be easily confused, so having all uppercase or all lowercase would avoid that. All lowercase has the PEP preference in the Python code while all those names of the MODFLOW family members, written in Fortran, are capitalized.

My preference would be all caps: NL are caps and MODFLOW are caps. The readability of the acronym seems a bit less important to me

**Comment by rubencalje** [comment]
I changed my vote from `NLmod` to `NLMOD`. I agree with the comment from @bdestombe, that writing the name in the same way as MODFLOW is nice.

**Comment by OnnoEbbens** [comment]
The final decision is we will use "NLMOD"
